.github/get-deps-mingw.ps1

```diff
@@ -0,0 +1,30 @@
+﻿$ErrorActionPreference = "Stop"
+$ProgressPreference = "SilentlyContinue"
+
+$sdlVersion = "2.0.10"
+$sdlImageVersion = "2.0.5"
+
+$sdlArhive = "SDL2-devel-$sdlVersion-mingw.tar.gz"
+$sdlImageArhive = "SDL2_image-devel-$sdlImageVersion-mingw.tar.gz"
+
+$sdlUrl = "https://www.libsdl.org/release/$sdlArhive"
+$sdlImageUrl = "https://www.libsdl.org/projects/SDL_image/release/$sdlImageArhive"
+
+"Downloading $sdlUrl" | Write-Host
+curl.exe -sSLfO $sdlUrl
+if ($LASTEXITCODE) { "Download failed" | Writ-Host: exit 1 }
+
+"Downloading $sdlImageUrl" | Write-Host
+curl.exe -sSLfO $sdlImageUrl
+if ($LASTEXITCODE) { "Download failed" | Writ-Host: exit 1 }

```

Comment:
I don't know PowerShell, but is this a typo? Should it be this:

```suggestion
if ($LASTEXITCODE) { "Download failed" | Write-Host: exit 1 }
```

(Additionally, if you do amend the commit for this, could you fix the typo of GitHub in the message?)

Reply:
Thank you, I did both of this, and I did the amend. Is the amend your preferred way of doing the reviews? I personally prefer another commit so it's clear in the PR what has been changed and then the maintainer (you) can squash commits he thinks deserve squashing @tmewett

Reply:
I prefer amending/rebasing for small changes (pressing "merge" in GitHub is optimal workflow for me :P) It is possible to compare between pushes by clicking the "force-pushed" in the PR update event, or whatever it's called